kroxylicious/src/main/java/io/kroxylicious/proxy/internal/filter/BuiltinFilterContributor.java

```diff
@@ -17,6 +17,7 @@ public class BuiltinFilterContributor extends BaseContributor<KrpcFilter> implem
     public static final BaseContributorBuilder<KrpcFilter> FILTERS = BaseContributor.<KrpcFilter> builder()
             .add("ApiVersions", ApiVersionsFilter::new)
             .add("BrokerAddress", BrokerAddressConfig.class, BrokerAddressFilter::new)
+            .add("FixedClientId", FixedClientIdFilter.FixedClientIdFilterConfig.class, FixedClientIdFilter::new)
```

Comment:
Is this something we really want to support in the default set of filters?

To me its something the Test source tree should be adding to the picture rather than the main sources.

Reply:
I'll see if we can wire up a TestFilterContributor in the integrationtests